### DISCUSSION TITLE
Sometimes we need support!

### DISCUSSION THREAD

**Discussion by gAlleb** [AzuraCast/AzuraCast#7497] [General]
Hello AzuraCast! Hey everyone!

Some of you might have noticed that Matthias (@Moonbase59) hasn't been around as much lately. He's going through a tough time right now and is facing some challenges in his life. He could really use our support.

Matthias has been a huge part of this community for years, volunteering his time and skills to make things better for everyone. He's always been there to help us out, so now it's our turn to show him some love.

If you're able to, please consider donating to help him out. Every little bit counts! You can donate through: 
> [!IMPORTANT]
> ### https://www.paypal.com/donate/?hosted_button_id=PBPR63362LDEU.
<hr>

He's also shared a recent update about his situation on his GitHub page. 
> [!IMPORTANT]
>### Please visit the link to read his latest entry [https://github.com/Moonbase59/autocue/blob/master/IMPORTANT.md](https://github.com/Moonbase59/autocue/blob/master/IMPORTANT.md)  
It gives a better understanding of what he's facing.
<hr>

Thank you for your generosity and for supporting our neighbor in need. **Thank you! ❤️**

**Comment by netbladenl** [comment]
Done! 
I really hope you find some light on your future patch, thanks for everything you've done so far for the community <3

**Reply by LordHelmchen666** [reply]
Also done. I hope some more people will help him.

**Comment by BusterNeece** [comment]
Just saw this earlier this morning. We'll be posting about it on our social channels as well. Really hoping things work out for him.

**Comment by Yahav** [comment]
Say no more.

**Comment by LordHelmchen666** [comment]
It seems he is sometimes active on github https://github.com/Moonbase59
Anyone have contact with him?
@Moonbase59 If you read this. Could you give us a short update. How are you? Could we help you further?
